easybuild/easyconfigs/k/kallisto/kallisto-0.43.1-foss-2016b.eb

```diff
@@ -14,8 +14,7 @@ toolchainopts = {'pic': True, 'usempi': True}
 
 source_urls = ['https://github.com/pachterlab/kallisto/archive/']
 sources = ['v%(version)s.tar.gz']
-
-checksums = ['2164938c2c61c04e338c4c132cf749f56d39e6f0b4c517121bca1fbc218e430e']
+checksums = ['7baef1b3b67bcf81dc7c604db2ef30f5520b48d532bf28ec26331cb60ce69400']
```

Comment:
Since there was no code change, should we retain the old checksum as a valid checksum?

```
checksums = [
    ('2164938c2c61c04e338c4c132cf749f56d39e6f0b4c517121bca1fbc218e430e',  # old release (diff. license)
     '7baef1b3b67bcf81dc7c604db2ef30f5520b48d532bf28ec26331cb60ce69400'),    # re-release with new license
]

```

Reply:
Good idea :+1:

Reply:
This feature is not implemented... Revert?

Reply:
It is implemented, see https://github.com/easybuilders/easybuild-framework/pull/2958

Reply:
Ah, it's the easyconfigs test that ensures SHA256 checksums are in place for everything that is failing, I can help with that...

Reply:
Hm, I guess this code would need to raise a deprecation warning then: https://github.com/easybuilders/easybuild-framework/pull/2958/files#diff-80e271b8fd31ff4cff170771374ccbe2R790-R791

Or the test for SHA256sum needs to be updated to allow this. Not sure what the "new" behavior is.

Reply:
Oh and https://github.com/easybuilders/easybuild-framework/pull/2958/files#diff-80e271b8fd31ff4cff170771374ccbe2R801-R802 is outdated too

Reply:
bug fixed in https://github.com/easybuilders/easybuild-framework/pull/3153